packages/ddex/processor/src/publishRelease.ts

```diff
@@ -206,19 +208,23 @@ export function prepareTrackMetadatas(release: DDEXRelease) {
         if (deal.audiusDealType == 'FollowGated') {
           const cond = { followUserId: release.audiusUser! }
           if (deal.forStream) {
+            meta.isStreamGated = true
```

Comment:
good catch on this.

relevant indexing code 
https://github.com/AudiusProject/audius-protocol/blob/b3a87c9f09bcc950b730f5e32aedf7e159cc8c77/packages/discovery-provider/src/tasks/entity_manager/entities/track.py#L684